trial-config-service/src/main/resources/application.properties

```diff
@@ -5,9 +5,7 @@ spring.jpa.hibernate.ddl-auto=none
 
 spring.hateoas.use-hal-as-default-json-media-type=false
 
-
 # Log properties in application.properties
-#logging.level.org.springframework.security=DEBUG
-
-# TODO ARTS-264 put these values into Azure Key Vault
-
+logging.level.org.springframework.security=DEBUG
+logging.level.org.hibernate.SQL=DEBUG
+logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
```

Comment:
Might want to tone these down a bit now you've got it working

Reply:
Done